packages/bpk-component-rating/src/BpkRating.scss

```
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins';
+
+.bpk-rating {
+  display: flex;
+
+  &--base-rating {
+    width: $bpk-spacing-sm * 4;
+    height: $bpk-spacing-sm * 4;
+    padding: $bpk-spacing-sm;
+    border-radius: $bpk-ratings-base-border-radius;
+    color: $bpk-color-white;
+    text-align: center;
+    vertical-align: middle;
+  }
+
+  &--text {
+    padding-left: $bpk-spacing-sm;
+  }
+
+  &--high-rating {
+    @include bpk-themeable-property( background-color, --bpk-rating-high-color, $bpk-color-green-600 );
```

Nitpick - we should use consistent spacing here (sadly we can't run this through Prettier as with JS code).

```suggestion
    @include bpk-themeable-property(background-color, --bpk-rating-high-color, $bpk-color-green-600);
```